pkg/user/delivery/http/users_proxy.go

```diff
@@ -11,32 +11,32 @@ import (
 	"github.com/CESARBR/knot-babeltower/pkg/user/entities"
 )
 
-// UserProxy represents the interface to the user's proxy operations
-type UserProxy interface {
+// UsersProxy represents the interface to the user's proxy operations
+type UsersProxy interface {
 	Create(user entities.User) (err error)
-	CreateToken(user entities.User) (string, error)
+	CreateUserToken(user entities.User) (token string, err error)
```

Comment:
Considering `UsersProxy` doesn't have `CreateAppToken` anymore, could we rename this method to just`CreateToken`? We know that's a user's token creation by the context.

Reply:
Yes! Gonna change it! :+1: :smiley:

Reply:
Okay, let me know when you update this PR.

Reply:
Done!